docs/antrea-network-policy.md

```diff
@@ -1062,6 +1065,37 @@ spec:
       - fqdn: "svcA.default.svc.cluster.local"
 ```
 
+## Node Selector 
+
+NodeSelector selects certain Nodes which match the label selector. Add Node IPs to address group memberSet.
+The following rule applied to Pods with label `app=antrea-test-app` and will `Drop` egress traffic to 
+Nodes which has labels `kubernetes.io/hostname=kind-control-plane`.
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: ClusterNetworkPolicy
+metadata:
+  name: egress-control-plane
+spec:
+  priority: 1
+  appliedTo:
+    - podSelector:
+        matchLabels:
+          app: antrea-test-app
+  egress:
+    - action: Drop
+      to:
+        - nodeSelector:
+            matchExpressions:
+              - key: kubernetes.io/hostname
+                operator: In
+                values:
+                  - kind-control-plane
```

Comment:
Maybe a more proper example for your use case is:
```
matchExpressions:
- key: node-role.kubernetes.io/control-plane
  operator: Exists
```
as `hostname` can only match a single host.

Reply:
Done.